fix(payments): add missing class for button styles to take effect

sardesam opened the pull request:
## Because

- Changes in button styling that are shared across FxA were not made in payments
- `button` element selector was removed from stylesheets in a separate commit, making all buttons and button-styled CTAs require `.button` class for styles to take effect

## This pull request

- Adds `.button` class to all button and button-styled CTAs
- Updates `index.scss` to target `.button` class and not `button` element
- Updates test to include `.button` class in submit button

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Other information (Optional)

Follow up to #13739 where `button` element was removed from shared stylesheets.

biancadanforth reviewed (changes requested):
Thanks Meghan! Do you see the button locally? I only see the top portion of it when I check this branch out.

![Screen Shot 2022-08-02 at 3 17 42 PM](https://user-images.githubusercontent.com/17437436/182455702-9cd09851-bdef-450f-8df6-54dacd3b762a.png)

LZoog commented:
> `button` element selector was removed from stylesheets and replaced with `.button` class in a separate commit, making all buttons and button-styled CTAs require this class for styles to take effect

FWIW I didn't actually _replace_ the style with `.button`, I just removed the previous selector targeting `button` _and_ `.button` to only target `.button`. However, I clearly did not do due diligence of checking SubPlat very well which shares content-server styles. 😔 Thanks for the fix! LGTM from a code standpoint but I'll let Bianca review locally since she's requested changes.

StaberindeZA commented:
> Thanks Meghan! Do you see the button locally? I only see the top portion of it when I check this branch out.
> 
> ![Screen Shot 2022-08-02 at 3 17 42 PM](https://user-images.githubusercontent.com/17437436/182455702-9cd09851-bdef-450f-8df6-54dacd3b762a.png)

I'm seeing the same issue on my end when testing locally.

sardesam commented:
Odd, I can see the button?

![button_fix_ss](https://user-images.githubusercontent.com/104124653/182461819-827f3286-fdc4-427f-9a7d-f8210106af1a.jpg)

Excuse the zoom, I wanted to get the full screen.

LZoog commented:
> I'm seeing the same issue on my end when testing locally.

Hm so `yarn build-css` should have been run automatically, but maybe payments' pm2 needs an update? Does running `yarn build-css` let you see it? We have #13817 filed that we're planning on throwing in the next maintenance sprint, probably good for you and @sardesam and @xlisachan to be aware of it even if it's not the issue here.

sardesam commented:
> FWIW I didn't actually _replace_ the style with `.button`, I just removed the previous selector targeting `button` _and_ `.button` to only target `.button`. However, I clearly did not do due diligence of checking SubPlat very well which shares content-server styles. 😔 Thanks for the fix! LGTM from a code standpoint but I'll let Bianca review locally since she's requested changes.

Thank you for clarifying/catching that. I updated the PR description 🙂 .

biancadanforth reviewed (commented):
Huh; I am in Firefox Nightly, and I noticed I needed to update the browser. After updating the browser, I do see it rendering correctly. Sorry for the confusion. @StaberindeZA : Is it possible you have the same issue?

I also noticed the avatar pic on the Sub Man page is not being sized properly...

![Screen Shot 2022-08-02 at 16 21 53-fullpage](https://user-images.githubusercontent.com/17437436/182466729-683b145c-ad81-4211-a727-a3f0ed520367.png)

sardesam commented:
Thanks @biancadanforth !
